moped-editor/src/queries/project.js

```diff
@@ -204,6 +204,12 @@ export const PROJECT_ACTIVITY_LOG = gql`
         user_id
       }
     }
+    activity_log_lookup_tables: moped_activity_log(
+      where: { record_project_id: { _eq: $projectId } }
+      distinct_on: record_type
+    ) {
+      record_type
+    }
```

Comment:
This is very cool! I did not know distinct_on was available in Hasura

Reply:
Same! I just happened across it when using GraphiQL in the Hasura console.